ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -983,6 +983,7 @@ private void setupTestState() throws InterruptedException, StoreException {
     // being picked for delete.
     time.sleep(Time.MsPerSec);
     long expectedStoreSize;
+    assertTrue("Expected empty store", store.isEmpty());
```

Comment:
you need to add a similar check in `CuratedLogIndexState`

Reply:
Done.